r/src/array_to_vector.cpp

```diff
@@ -418,6 +418,7 @@ class Converter_Struct : public Converter {
   std::vector<std::shared_ptr<Converter>> converters;
 };
 
+// Shouldn't this cast before dividing? Otherwise we're doing integer division
```

Comment:
Depends what you want, integer division is much faster than floating division.

Reply:
I would think we'd be more interested in fidelity of the data transformation than raw speed. Assuming I understand what the operators are doing, isn't `float(1234 / 1000)` a lot different from `float(1234) / 1000`?

Reply:
Correct, you lose the fractional part, but yes we can revise that. I have no preference, it's probably better to preserve precision.

Reply:
I made the change and tests pass, so let's go with it.